src/features/unified-share-modal/__tests__/SharedLinkAccessMenu.test.js

```diff
@@ -62,6 +62,21 @@ describe('features/unified-share-modal/SharedLinkAccessMenu', () => {
             });
             expect(sharedLinkAccessMenu).toMatchSnapshot();
         });
+
+        test('should render tooltips for access level menu items if disabled by malicious content', () => {
+            const sharedLinkAccessMenu = getWrapper({
+                accessLevelsDisabledReason: {
+                    peopleInYourCompany: 'malicious_content',
+                    peopleWithTheLink: 'malicious_content',
+                },
+                allowedAccessLevels: {
+                    peopleInThisItem: true,
```

Comment:
Whom does `peopleInThisItem` refer to -- owners of the item?

Reply:
I think this is equivalent to collaborators only in the sharing options